src/main.py

```diff
@@ -15,14 +15,18 @@
 
 
     argv_len = len(sys.argv)
-
+    src_path = os.path.dirname(os.path.realpath(__file__))
+    
     if argv_len == 1:
-        src_path = os.path.dirname(os.path.realpath(__file__))
         pylarm(src_path)
         
     elif argv_len == 2:
         try:
-            print(arguments[ '2' ][ sys.argv[1] ][ 'info' ])
+            # passed argument is in cli_options.json
+            if sys.argv[ 1 ] in arguments[ '2' ]:
+                print(arguments[ '2' ][ sys.argv[1] ][ 'info' ])
+            else:
+                pylarm(src_path, given_time=sys.argv[1])
```

Comment:
This line right here allows options such as `foo` or `--foo` for example pass through and are attempted to become parsed as a time in `pylarm.py`

What originally would happen if you were to put in an invalid option is fall to one of the `excepts` and print our an error message called from `print_cli_options_error(sys.argv)`.